jaxley/channels/__init__.py

```diff
@@ -3,5 +3,7 @@
 
 from jaxley.channels.channel import Channel  # isort: skip
 from jaxley.channels.hh import HH
-from jaxley.channels.lif import LIF
+from jaxley.channels.non_capacitive.izhikevich import Izhikevich
```

Comment:
I get why you group them in `non_cap`, which I think is fine, but I think splitting this into `channels.rate` and `channels.spike` submodules would arguably be a bit nicer.